test/utils.js

```diff
@@ -0,0 +1,23 @@
+const mochaAsync = fn => {
+  return done => {
+    fn.call().then(done, err => {
+      done(err);
+    });
+  };
+};
+
+const removeSpace = text => {
+  return text.replace(/\s/g, '');
+};
+
+const removeTags = html => {
+  return html.replace(/<\/?[^>]+(>|$)/g, '');
+};
+
+const createRandomString = () => {
+  return Math.random()
```

Comment:
Maybe set the random seed outside of this function so that although the strings are random they are consistent across tests?